src/__swaps__/utils/swaps.ts

```diff
@@ -380,11 +380,11 @@ export const DEFAULT_SLIPPAGE_BIPS = {
   [ChainId.blast]: 200,
 };
 
-export const slippageInBipsToString = (slippageInBips: number) => (slippageInBips / 100).toString();
+export const slippageInBipsToString = (slippageInBips: number) => (slippageInBips / 100).toFixed(1);
```

Comment:
Should this use safe math?

Reply:
i don't think this will ever be able to overflow, so it's pretty safe to not.